### DISCUSSION TITLE
WHERE IS MY LANGUAGE???

### DISCUSSION THREAD

**Discussion by amogus141** [ScratchAddons/ScratchAddons#3511] [Translation]
scratch addons translate
**vietnamese**

**Comment by apple502j** [comment]
Scratch Addons is not currently translated into Vietnamese.

**Comment by Dhuls-Scratch** [comment]
If you want to help with a Vietnamese translation, please go to https://www.transifex.com/scratch-addons/scratch-addons-website/

**Reply by amogus141** [reply]
owo thanks
